### PR TITLE
add some components 

### PR DESCRIPTION
**Pull request by ghanane**

### CODE REVIEW

**Review by xdegenne — CHANGES_REQUESTED** [review]
You shouldn't commit inputs.json files since they can contain credentials, we prefer commit inputs.json.tpl if needed. You shouldn't commit ansible .retry files (this kind of file are ansible execution artifacts)

File: org/alien4cloud/automation/a4c/webapp/ssl/ca.pem [hunk 1]
```diff
@@ -1,35 +1,32 @@
 -----BEGIN CERTIFICATE-----
-MIIF/TCCA+WgAwIBAgIJAOPMOedozXvGMA0GCSqGSIb3DQEBCwUAMFwxCzAJBgNV
```
Comment (xdegenne) [inline org/alien4cloud/automation/a4c/webapp/ssl/ca.pem:2]: Why this key has been changed ?

File: org/alien4cloud/automation/a4c/webapp/ssl/ca-key.pem [hunk 2]
```diff
@@ -1,54 +1,54 @@
 -----BEGIN RSA PRIVATE KEY-----
```
Comment (xdegenne) [inline org/alien4cloud/automation/a4c/webapp/ssl/ca-key.pem:1]: Why this key has been changed ?

File: org/alien4cloud/automation/a4c/webapp/bin/alien.sh [hunk 3]
```diff
@@ -1,38 +1,32 @@
 #!/bin/bash
+
 APP_NAME=alien4cloud
-PIDFILE=/var/run/${APP_NAME}.pid
-ALIEN_PATH="/opt/alien4cloud/alien.war"
-ALIEN_CONF="/etc/alien4cloud/"
+#PIDFILE=/var/run/${APP_NAME}.pid
+PROC=`ps -ef | grep -e alien4cloud* | grep -v grep | awk -F " " '{ print $2 }'`
 
-JAVA_OPTIONS="-server -showversion -XX:+AggressiveOpts -Xmx1400m -Xms1400m -XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:+HeapDumpOnOutOfMemoryError -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintGCApplicationStoppedTime -Xloggc:/var/log/alien4cloud/gc.log ${JVM_ARGS}"
```
Comment (xdegenne) [inline org/alien4cloud/automation/a4c/webapp/bin/alien.sh:7]: ${JVM_ARGS} has disarpeared , why ?

File: org/alien4cloud/automation/a4c/config/backupRestoreS3/scripts/restore.sh [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+# for line in $(cat /home/ubuntu/.aws/config) ; do
+#   if [ "$line" != "[default]" ] ; then
+#   export $line
+#   fi
+# done
+
+user=$(whoami)
+
+# script to restore elasticsearch and alien4cloud
+A4C_CONFIG="/opt/alien4cloud/alien4cloud/config/alien4cloud-config.yml"
```
Comment (xdegenne) [inline org/alien4cloud/automation/a4c/config/backupRestoreS3/scripts/restore.sh:13]: Non on ne peut pas mettre cela en dur. Si ça sert uniquement à récupérer l'IP de ES, pourquoi ne pas avoir de relation vers le endpoint ES

File: org/alien4cloud/automation/a4c/config/backupRestoreS3/scripts/restore.sh [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+ES_IP=$(cat ${A4C_CONFIG} | grep "hosts: " | grep -oE '((1?[0-9][0-9]?|2[0-4][0-9]|25[0-5])\.){3}(1?[0-9][0-9]?|2[0-4][0-9]|25[0-5])')
+
+# Initiate ES
+ES_SNAPSHOT_DIR=${ES_SNAPSHOT}
+ES_SNAPSHOT_NAME=${ES_SNAPSHOT}
+
+# setup elastic search snapshot repository
+echo "initiate snapshot repo"
+
+curl -s -XPUT "${ES_IP}:9200/_snapshot/es_backup/" -d '{
```
Comment (xdegenne) [inline org/alien4cloud/automation/a4c/config/backupRestoreS3/scripts/restore.sh:25]: L'ip du ES en dur, pareil, je ne pense pas que cette IP soit invariable

**Review by xdegenne — CHANGES_REQUESTED** [review]

File: org/alien4cloud/automation/a4c/webapp/ssl/ca.pem (hunk 1 above)
Comment (ghanane) [reply org/alien4cloud/automation/a4c/webapp/ssl/ca.pem:2]: coz the certficate was expired

**Pull request closed without merging**